<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileManager4</class>
 <widget class="QWidget" name="ProfileManager4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Manager Gen 4</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonNew">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonEdit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="TableView" name="tableView"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>ProfileManager4</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
